cli/src/main/resources/nls/Help_de.properties

```
@@ -80,7 +80,7 @@ cmd.pgadmin=Werkzeug Kommando für pgAdmin.
 cmd.pgadmin.detail=pgAdmin ist ein Open-Source-Verwaltungswerkzeug for PostgreSQL und verwandte relationale Datenbanken. Detaillierte Dokumentation ist zu finden unter https://www.pgadmin.org/docs/pgadmin4/latest/index.html
 cmd.quarkus=Werkzeug Kommando für Quarkus (Framework für Cloud-native Anwendungen).
 cmd.quarkus.detail=Quarkus ist ein Kubernetes-native Java-Framework zur Entwicklung von Cloud-native Anwendungen. Detaillierte Dokumentation ist zu finden unter https://quarkus.io/
-cmd.repository=Richtet das vorkonfigurierte Git Repository ein.
+cmd.repository=Richtet das vorkonfigurierte Git Repository ein. 'ide repository setup <repository>'.
```

As a native German, I can slightly improve your translation ;)
```suggestion
cmd.repository=Richtet das vorkonfigurierte Git Repository ein mittels 'ide repository setup <repository>'.
```